packages/runtime-dom/src/helpers/useCssVars.ts

```diff
@@ -80,7 +80,9 @@ function setVarsOnNode(el: Node, vars: Record<string, string>) {
   if (el.nodeType === 1) {
     const style = (el as HTMLElement).style
     for (const key in vars) {
-      style.setProperty(`--${key}`, vars[key])
+      if (vars[key] || vars[key] === '') {
```

Comment:
```suggestion
      if (vars[key] !== undefined && vars[key] !== null) {
```

Consider [this example](https://deploy-preview-7475--vue-sfc-playground.netlify.app/#eNqFkk9LxDAQxb/KkMsqmNSDItS6IIJnwWsutTvVSjoTknRlWfa7O0n3jwjLnsIkb+ZNf69b9ey9WU+oatXELgw+QcQ0+aWlYfQcErzw6KEPPMLCVLnI8oUlAEsdU0zgA/sIT7DCfiB8y9XVNgs6dhxqeE9hoE9Lu+tHS00124iBFAlH79qEuUpNsVq3bsInq3pmq5avzE1xPSmWdLoCmJvqQxdNzklbPs6L+tZFFFU5z8tuRXJ7fG6q47LqRs1w9Nh68x2ZBF/5Yrt/iFbVUG7ynQDLtVVfKflYV9UKveON9gHXA/7oh7uHe61FpWPfafHYfAaeaGUIkxv6jWm9r+TZhInSMKLBOOqPwD8Rg9hblY12AlgWOyR0PtDt36Bgt892DvVCoIVMDQXyhTiFpofOtTEKyU6WYkJKQnSG0kTHaSnd+Sjq6h/kMjCmjZunmeOMA1etI4+YvuTfqmGtPwZaXS3K6qbsucj7FS5iU+ao3S+fTASh)